src/core/ext/client_channel/client_channel_factory.c

```diff
@@ -33,41 +33,41 @@
 
 #include "src/core/ext/client_channel/client_channel_factory.h"
 
-void grpc_client_channel_factory_ref(grpc_client_channel_factory* factory) {
+void grpc_client_channel_factory_ref(grpc_client_channel_factory *factory) {
```

Comment:
The position of the * is still inconsistent with the rest of core, but yes, this is not the place to make those changes.

Reverted.